### PR TITLE
Remote and Local timestamps

### PR DESCRIPTION
**Pull request by JPonte**

### CODE REVIEW

**Comment by someoneEsle** [comment]
- [ ] Bug 1
Messages are marked as edited even if they are not

File: app/src/main/scala/com/waz/zclient/messages/MessagesController.scala [hunk 1]
```diff
@@ -20,15 +20,15 @@ package com.waz.zclient.messages
...
+import com.waz.model._
 import com.waz.service.ZMessaging
 import com.waz.utils.events.{EventContext, EventStream, Signal}
 import com.waz.zclient.controllers.navigation._
 import com.waz.zclient.conversation.ConversationController
 import com.waz.zclient.pages.main.conversationpager.controller.{ISlidingPaneController, SlidingPaneObserver}
 import com.waz.zclient.utils.ContextUtils
 import com.waz.zclient.{Injectable, Injector, WireContext}
-import org.threeten.bp.Instant
+import com.waz.utils.RichWireInstant
```
Comment (CalumMcCall) [inline app/src/main/scala/com/waz/zclient/messages/MessagesController.scala]: Is this import necessary?

File: app/src/main/scala/com/waz/zclient/sharing/ShareToMultipleFragment.scala [hunk 2]
```diff
@@ -53,6 +53,7 @@ import com.waz.zclient.ui.views.CursorIconButton
 import com.waz.zclient.usersearch.views.{PickerSpannableEditText, SearchEditText}
 import com.waz.zclient.utils.ContextUtils.{getDimenPx, showToast}
 import com.waz.zclient.utils.{RichView, ViewUtils}
+import com.waz.utils.RichWireInstant
```
Comment (CalumMcCall) [inline app/src/main/scala/com/waz/zclient/sharing/ShareToMultipleFragment.scala]: Seems unused?

File: app/src/main/scala/com/waz/zclient/notifications/controllers/CallingNotificationsController.scala [hunk 3]
```diff
@@ -41,7 +41,7 @@ import com.waz.zclient.common.views.ImageController
 import com.waz.zclient.utils.ContextUtils._
 import com.waz.zclient.utils.{DeprecationUtils, RingtoneUtils}
 import com.waz.zms.CallWakeService
-import org.threeten.bp.Instant
+import com.waz.utils._
```
Comment (CalumMcCall) [inline app/src/main/scala/com/waz/zclient/notifications/controllers/CallingNotificationsController.scala]: Is this needed?

File: app/src/main/scala/com/waz/zclient/messages/RecyclerCursor.scala [hunk 4]
```diff
@@ -22,15 +22,15 @@ import com.waz.ZLog._
...
+import com.waz.model.{ConvId, LocalInstant, MessageData}
 import com.waz.service.ZMessaging
 import com.waz.service.messages.MessageAndLikes
 import com.waz.threading.Threading
 import com.waz.utils._
 import com.waz.utils.events.{EventContext, Signal, Subscription}
 import com.waz.zclient.messages.RecyclerCursor.RecyclerNotifier
 import com.waz.zclient.{Injectable, Injector}
-import org.threeten.bp.Instant
+import com.waz.utils.RichWireInstant
```
Comment (CalumMcCall) [inline app/src/main/scala/com/waz/zclient/messages/RecyclerCursor.scala]: unused?

**Pull request merged**